### PR TITLE
Add print_keys script

### PR DESCRIPTION
**Pull request by madninja**
This adds a miner/bin/print_keys shell script that prints out a list of terms for onboarding/pub keys. 

To get this to work the miner_sup startup for key retrieval was moved into a new miner_keys module

### CODE REVIEW

File: scripts/print_keys [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+erl -pa {{lib_dirs}}/**/ebin -noshell -eval 'miner_keys:print_keys("{{sys_config}}")' -eval 'init:stop()'
```
Comment (Vagabond) [inline scripts/print_keys]: Why not use a miner subcommand that doesn't do a rpc call?

Comment (madninja) [reply scripts/print_keys]: Because I was lazy and this was quicker?

Comment (madninja) [reply scripts/print_keys]: I’ll get that fixed up

File: public_keys.config [hunk 2]
```diff
@@ -0,0 +1,2 @@
+{pubkey,"112YR4mqBrc6DxrVonKsYE6bb6TGab3K1vVfpF8yAebRSd5YgZDy"}.
```
Comment (Vagabond) [inline public_keys.config]: What is this file for?

Comment (madninja) [reply public_keys.config]: accidental checkin

**Comment by Vagabond** [comment]
@fvasquez we should call this in the runit init script in nextgate and write the output to a file

**Pull request merged**